### PR TITLE
refactor: worked on ViewMyBookActivity Firebase integration

### PR DESCRIPTION
**Pull request by aakindel**
The basic setup for getting a book's information from Firebase and into the app is mostly functional.

### CODE REVIEW

File: app/src/main/java/com/example/pocketbook/activity/ViewMyBookActivity.java [hunk 1]
```diff
@@ -21,8 +36,42 @@ protected void onCreate(Bundle savedInstanceState) {
...
         ViewPager viewPager = findViewById(R.id.viewMyBookViewPager);
+
+        Bundle extras = getIntent().getExtras();
+        bookID = (extras == null) ? null : extras.getString("FIRESTORE_BOOK_UID");
+
+        mAuth = FirebaseAuth.getInstance();
+        email = "test@pocketbook.com";
+        password = "password5";
+
+        mAuth.signInWithEmailAndPassword(email, password)
```
Comment (thefreezer) [inline app/src/main/java/com/example/pocketbook/activity/ViewMyBookActivity.java:47]: Will this call be removed once the Login page is setup?

**Review by MahmoodFalmaz — APPROVED** [review]
Approved

**Review by Head8che — APPROVED** [review]
Approved

**Review by Head8che — CHANGES_REQUESTED** [review]
Pending

**Review by Head8che — APPROVED** [review]
Approved

**Pull request merged**